### DISCUSSION TITLE
Why do we need start snoozed?

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#3345] [Q&A]
If you look under Alert Settings, you will see an option called "Start Snoozed".  

![Screenshot_20240215-125637](https://github.com/NightscoutFoundation/xDrip/assets/51497406/dc56f16f-1262-4274-b8f7-059c90e829c4)  
<br/>    
  
What is the purpose of this setting?
If you think you need it, please explain why.

Thanks

**Comment by Navid200** [comment]
Let's say we have the snooze time for the low alert at the default 35 minutes.
Let's say the readings are as shown in the following image.
![StartSnoozed](https://github.com/NightscoutFoundation/xDrip/assets/51497406/7d198d32-3afe-40d2-bc74-d059ed19b61e)


So, the readings cross the low threshold for the first time at 8:00.  But, because start snoozed is enabled, it will not trigger the alert.
Since the snooze time is set to 35 minutes, the first time the alert will go off will be at 8:35.
Am I right?

What if the snooze was set to 2 hours?  In that case, the alert will not go off until 10:00, right?

**Comment by raulvelasquez101** [comment]
I see it as a way to avoid unnecessary alarms for compression lows at night. In my experience, compression lows don't tend to last long, if they occur while I'm sleeping. If a low persists for X amount of time, then I'm more certain that it isn't a compression low, and I would want an alarm to wake me up and correct it, vs. me waking up multiple times in the night for compression lows.
